### PR TITLE
aktualizr-lite-public-stream: drop recipe

### PR DESCRIPTION
**Pull request by quaresmajose**
Improves the build speed with `INHIBIT_DEFAULT_DEPS` and reduce the space required with `PACKAGE_ARCH=all`

### CODE REVIEW

File: meta-lmp-base/recipes-sota/config/aktualizr-lite-public-stream.bb [hunk 1]
```diff
@@ -10,18 +10,21 @@ SRC_URI = "file://10-lite-public-stream.toml.in"
...
 do_install() {
 	install -m 0700 -d ${D}${libdir}/sota/conf.d
-	install -m 0644 ${WORKDIR}/10-lite-public-stream.toml ${D}${libdir}/sota/conf.d/10-lite-public-stream.toml
+	install -m 0644 ${WORKDIR}/10-lite-public-stream.toml.in \
+		${D}${libdir}/sota/conf.d/10-lite-public-stream.toml
+
+	sed -i -e 's/@@AKLITE_TAG@@/${AKLITE_TAG}/' \
+		${D}${libdir}/sota/conf.d/10-lite-public-stream.toml
 }
+do_install[vardeps] += "AKLITE_TAG"
```
Comment (doanac) [inline meta-lmp-base/recipes-sota/config/aktualizr-lite-public-stream.bb:28]: We might be able to drop this entire recipe. I don't think it's possible to use this anymore.

Comment (quaresmajose) [reply meta-lmp-base/recipes-sota/config/aktualizr-lite-public-stream.bb:28]: When I am on it I do a search in the layers and this recipe is not used by any other in public LmP, but our CI sstate cache  have this installed in some `lmp-base-console-image` that's what alerted me

Comment (quaresmajose) [reply meta-lmp-base/recipes-sota/config/aktualizr-lite-public-stream.bb:28]: It is installed by ci-script
```
|  # Bitbake custom logconfig
|  BB_LOGCONFIG = "bb_logconfig.json"
|  IMAGE_INSTALL:append = " aktualizr-lite-public-stream"
|  # Take a tag from a spec like:
|  #  https://docs.foundries.io/latest/reference-manual/ota/advanced-tagging.html
|  # and find the first tag name to produce a sensible default
|  LMP_DEVICE_REGISTER_TAG = "postmerge"
|  LMP_DEVICE_FACTORY = "lmp"
|  LMP_DEVICE_API = "https://api.foundries.io/ota/devices/"
|  SOTA_PACKED_CREDENTIALS = ""
|  DISTRO_VERSION:append = "-1850-88-49-g03de33b"
```
https://ci.foundries.io/projects/lmp/builds/1850/intel-corei7-64/artifacts/console.log

Comment (quaresmajose) [reply meta-lmp-base/recipes-sota/config/aktualizr-lite-public-stream.bb:28]: maybe @rsalveti know if it is still necessary https://github.com/foundriesio/ci-scripts/commit/76f9596a83856c48a6ca9a6a13bc6fac4c0a1d98

Comment (ricardosalveti) [reply meta-lmp-base/recipes-sota/config/aktualizr-lite-public-stream.bb:28]: Yes, we can drop this recipe and the related ci-scripts change. We don't publish targets anymore, so fine to drop.

Comment (quaresmajose) [reply meta-lmp-base/recipes-sota/config/aktualizr-lite-public-stream.bb]: droped in the last revision.

**Review by ricardosalveti — APPROVED** [review]
LGTM

**Pull request merged**